softdevice_controller/CHANGELOG.rst

```diff
@@ -9,18 +9,19 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
-Master branch
+v1.5-branch
```

Comment:
```suggestion
nRF Connect SDK v1.5.1
```
I don't think it makes sense to mention the v1.5 branch here - we don't tell users to use version branches (only tags or master).
(Also add *** as needed in the next line.)